OSDOCS-5131: arches and dev preview nit change (#55431)
openshift/openshift-docs: modules/microshift-install-system-requirements.adoc

* You have a subscription that includes {product-title} RPMs.
* You have a Logical Volume Manager (LVM) Volume Group (VG) with sufficient capacity for the Persistent Volumes (PVs) of your workload.

> .{product-title} supports the following implementations of a 64-bit version of the x86 architecture.

Reviewer: Do we say anything like this about OCP? How is it phrased there?

I think the original point was that both of the entries in the table below are effectively the same. They're different manufacturers of x86 architecture chips, but we don't produce different builds for them. Do we need the table at all? For example, on https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/installing/installing-preparing#supported-installation-methods-for-different-platforms I see x86_64 on some of the cloud platforms. Should we just say that inline here and drop the table entirely?

```suggestion
.{product-title} supports systems based on the x86_64 architecture.
```

Author: The SSG provides guidance for reference to architectural references. https://redhat-documentation.github.io/supplementary-style-guide/#_a, if you scroll down to the AMD64 entry.

Correct, I understand these are different brands of the same thing. 

We can remove the table if it will not be needed for future docs. Are we adding support in the near future for just one more architecture types or several?

Reviewer: > The SSG provides guidance for reference to architectural references. https://redhat-documentation.github.io/supplementary-style-guide/#_a, if you scroll down to the AMD64 entry.

OK, let's use that then.

> Correct, I understand these are different brands of the same thing.

OK, I wasn't sure if we needed both because we want to mention both brands. I think the table is the reason the person opened the ticket originally.

> We can remove the table if it will not be needed for future docs. Are we adding support in the near future for just one more architecture types or several?

We will support 64-bit ARM systems in 4.13. Unfortunately I don't see that in the style guide. We usually call that arm64 or aarch64 in the code.

It's up to you whether we need the table. It's not _wrong_, but feels a little redundant right now. If it's better to have it and then expand it later, that makes sense and I agree with keeping it.

Author: I took that and the OS table out, simplified. We might use a small bulleted list for easy reading later as we add things.
